README.md

```diff
@@ -23,27 +23,29 @@ pip3 install openai
 
 Next, you need to give these scripts access to the OpanAI API token. This is possible in one of two ways:
 
-### 1. With a `tokens.py` file
+### 1. With a `translate_info.py` file
 
-In the root of this repository, create a file called `tokens.py` which contains this line of code:
+In the root of this repository, create a file called `translate_info.py` which contains your constraints:
 
 ```python
 CHATGPT_TOKEN = "put the token from OpenAI here"
+
+APP_CONTEXT = "This is my App, here is a summary what it does." # optional
 ```
 
 ### 2. Via Environment Arg
 
 Start the python script as follows
 
 ```bash
-OPENAI_API_KEY="put the token from OpenAI here" python3 filename.py
+OPENAI_API_KEY="put the token from OpenAI here" python3 translate_info.py
```

Comment:
`translate_info.py` should not be executed, only the other files

Reply:
oh sorry, I'm not that familiar with Python and how environmental variables work, gonna restore it now